build.gradle

```diff
@@ -98,7 +98,6 @@ subprojects {
 
         testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junitJupiterVersion"
         testRuntimeOnly 'org.junit.platform:junit-platform-launcher:1.4.0'
-        testRuntimeOnly 'org.slf4j:slf4j-nop:1.7.25'
```

Comment:
I think we had this for Spark. It was VERY important to remove this as DropWizard fails on startup if there are multiple SLF4J bindings present.  I added a warning note to the http-server README about this.